compiler/env/OMRCPU.cpp

```diff
@@ -100,8 +100,10 @@ OMR::CPU::initializeByHostQuery()
 bool
 OMR::CPU::supportsFeature(uint32_t feature)
    {
+   if (TR::Compiler->omrPortLib == NULL)
+      return false;
```

Comment:
If I understand correctly following this change we will never return `true` in OMR at the moment because the port library is not initialized. Thus we have switched from dynamically determining the processor in the codegen (via `cpuid`, etc.), to not doing so.

Will that result in us generating different assembly sequences?

Reply:
Will that result in us generating different assembly sequences? Yes

Yeah, that's an issue I'm going to address right now.

I was assuming that except OpenJ9, nothing else is relying on the processor detection features, that's why I was planning to return `false` for everything.

So here's a new plan:

- if there is no port lib initialized, we use the old ways to determine processor
- if there is a port lib initialized, we use the port lib and my new APIs

Do you think this is a feasible and acceptable plan?

Reply:
That seems reasonable as long as the cruft is nicely tucked away in a single place so we can nuke it once we can initialize the port library in OMR compiler.

Reply:
Okay cool, I'll take a look to see if that's possible.